tools/ci_connector_ops/ci_connector_ops/pipelines/builds/normalization.py

```diff
@@ -6,24 +6,14 @@
 from ci_connector_ops.pipelines.actions import environments
 from ci_connector_ops.pipelines.bases import Step, StepResult, StepStatus
 from ci_connector_ops.pipelines.contexts import ConnectorContext
-from ci_connector_ops.utils import Connector
 from dagger import Container
 
 
+# TODO this class could be deleted
+# if java connectors tests are not relying on an existing local normalization image to run
```

Comment:
I think this is partially tracked in https://github.com/airbytehq/airbyte/issues/24880 ! (would also need to do the other 9 normalized destinations, which is still pending a decision)

unless there are other tests that also depend on the normalization image, which I'm not aware of

Reply:
or maybe not, depending on how we implement it. Forgot that we might end up just running the existing pytest stuff directly, which does depend on the normalization docker image